scripts/link-check-git-diff.sh

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 set -euxo pipefail
-$(dirname "$0")/link-check.sh <(git diff origin/master -U0)
+$(dirname "$0")/link-check.sh <(git diff $(git merge-base HEAD origin/master) -U0 | grep '^\+')
```

Comment:
Maybe you can just use `lint-staged` (used/configured in package.json) to sent the staged files into this script @casperdcl ?

Reply:
Can't because that would affect the pre-commit hook

Reply:
(and we can't assume devs have the prereqs to run the checks)